.env.example

```
@@ -1,7 +1,11 @@
 PORT=4000
-MONGO_URI=mongodb://localhost:27017/conciApi #excel-pilot
+MONGO_URI=mongodb://localhost:27017/excelPilot
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MONGO_URI key should go before the PORT key

(UnorderedKey)